### PR TITLE
Navigation to beerDetail fragment

### LINKED ISSUE #52: Navigation error when opening beer details card, not able to return to home fragment

### PR DESCRIPTION
**Pull request by alessiodicorato**

### CODE REVIEW

File: app/src/main/java/co/develhope/chooseyourownbeer/BeerDetail.kt [hunk 1]
```diff
@@ -11,28 +11,42 @@ import co.develhope.chooseyourownbeer.model.Beer
...
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
         _binding = BeerDetailBinding.inflate(inflater, container, false)
-        val beerId= arguments?.getLong("BEER_ID") ?: 0
-        val beer = getBeerFromId(beerId)
-        getBeerFromId(beerId)?.let { setupUI(it) }
+        // val beerId = arguments?.getLong("BEER_ID") ?: 0
+        val beer = beerId?.let { getBeerFromId(it) }
```
Comment (vincenzocostagliola) [inline app/src/main/java/co/develhope/chooseyourownbeer/BeerDetail.kt]: troppi let, userei solo il secondo per controllare se ho effettivamente l'oggetto o meno.

File: app/src/main/java/co/develhope/chooseyourownbeer/BeerDetail.kt [hunk 2]
```diff
@@ -11,28 +11,42 @@ import co.develhope.chooseyourownbeer.model.Beer
...
+        beer?.let { setupUI(it) }
```
Comment (vincenzocostagliola) [inline app/src/main/java/co/develhope/chooseyourownbeer/BeerDetail.kt]: Nel caso in cui l'oggetto non fosse trovato che succede? Direi di mostrare una textview con un messaggio di errore

**Comment by alessiodicorato** [comment]
[Gear icon instead of "SETTINGS" text in profile fragment](https://trello.com/c/XzdEx5Iz/23-gear-icon-instead-of-settings-text-in-profile-fragment)

**Comment by alessiodicorato** [comment]
[Navigation error when opening beer details card, not able to return to home fragment](https://trello.com/c/2i9M80vs/24-navigation-error-when-opening-beer-details-card-not-able-to-return-to-home-fragment)

**Comment by alessiodicorato** [comment]
[BeerDetail - companion object](https://trello.com/c/DJBvBjjH/2-beerdetail-companion-object)

**Comment by alessiodicorato** [comment]
[BeerDetail - setupUi](https://trello.com/c/zacPbK6G/3-beerdetail-setupui)

**Comment by alessiodicorato** [comment]
[BeerDetail - fun setupUI](https://trello.com/c/I1OVExUy/4-beerdetail-fun-setupui)

**Comment by alessiodicorato** [comment]
[A Long instead of Int?](https://trello.com/c/H7TAaXjO/1-a-long-instead-of-int)

File: app/src/main/java/co/develhope/chooseyourownbeer/BeerDetail.kt (hunk 2 above)
Comment (alessiodicorato) [reply app/src/main/java/co/develhope/chooseyourownbeer/BeerDetail.kt]: Anche un Toast andrebbe bene o consigli la TextView?

**Comment by alessiodicorato** [comment]
[SearchFragment - onAdapterCLick](https://trello.com/c/7aXVOSOK/16-searchfragment-onadapterclick)

**Comment by alessiodicorato** [comment]
[beer_detail.xml](https://trello.com/c/qRa2Zarm/15-beerdetailxml)

**Comment by alessiodicorato** [comment]
https://user-images.githubusercontent.com/85505676/171810753-dc710c46-b6ef-4d2b-8ca6-183c2fa12a42.mp4

**Comment by alessiodicorato** [comment]
[HomeFragment - onAdapterCLick](https://trello.com/c/janC8k7o/9-homefragment-onadapterclick)

**Comment by alessiodicorato** [comment]
[HomeFragment - OnCreateView](https://trello.com/c/RqcHly7g/8-homefragment-oncreateview)

**Comment by alessiodicorato** [comment]
https://user-images.githubusercontent.com/85505676/171812167-6a0d2ce0-eb02-4fda-812c-7e67ba9a39bd.mp4

Video shows the new navigation to BeerDetailActivity and the new gear icon instead of "Settings" in profile page

**Comment by alessiodicorato** [comment]
[SearchFragment - printCount](https://trello.com/c/MKj3xiEp/30-searchfragment-printcount)

**Review by GiusyMancuso — APPROVED** [review]

**Pull request merged**